Taskfile.yml

```diff
@@ -158,3 +158,14 @@ tasks:
     desc: Accept all test snapshots. (Shortcut to accept snapshots after running tests)
     cmds:
       - Rscript -e 'testthat::snapshot_accept()'
+
+  fmt:
+    desc: Format all R and Rust code.
+    sources:
+      - "{{.R_SOURCE}}"
+      - "{{.RUST_SOURCE}}"
+      - tests/**/*
+    cmds:
+      - air -V | grep -q "0.4.0" || exit 1
```

Comment:
Only way I found to ensure `task fmt` errors if `air` is not at the right version

Reply:
Perhaps it is the design of Taskfile to set `preconditions` here, but in any case, it is not common to fix the version of VS Code extensions, so for now, it would be better to assume automatic updates of the latest version?

https://taskfile.dev/usage/#using-programmatic-checks-to-cancel-the-execution-of-a-task-and-its-dependencies